webdriver/tests/bidi/web_extension/uninstall/uninstall.py

```diff
@@ -6,8 +6,8 @@
 async def test_uninstall(bidi_session, extension_data):
     web_extension = await bidi_session.web_extension.install(
         extension_data={
-            "type": "base64",
-            "value": extension_data["base64"],
+            "type": "path",
+            "path": extension_data["path"],
```

Comment:
@whimboo looks like this does not work for Firefox?

Reply:
Hm, so with your change we actually see a file system access error. Could it be that the first uninstall removed the folder?

Originally we used base64 here to also allow testing uninstall on mobile. There is no way to transfer files yet and this is the only mode. Maybe you could parameterize the method so that we cover all three types?

Reply:
I am not sure if pytest allows using fixtures in parameters? 

> Could it be that the first uninstall removed the folder?

Does Firefox do it? it would be unexpected if the folders are deleted.

I will revert this change for now.